nio/responses.py

```diff
@@ -565,6 +590,20 @@ def from_dict(
         return cls(members, room_id)
 
 
+@attr.s
+class JoinedRoomsResponse(Response):
```

Comment:
These need docs. I know that this file isn't documented well but we want to increase the number of docs instead of decrease.